components/rmrk/Gallery/GalleryItem.vue

```diff
@@ -261,6 +262,18 @@ export default class GalleryItem extends mixins(PrefixMixin) {
     this.priceChartData = data
   }
 
+  @Debounce(500)
+  private async updateEventList() {
+    const { data } = await this.$apollo.query<{ nftEntities }>({
+      client: 'subsquid',
+      query: nftByIdMinimal,
```

Comment:
Good job sir,

but why do you query list of NFTs instead of just single nft with events?

Reply:
You are right. Because I'm not that familiar with graphql query : )

Reply:
Kjut lovely ❤️